sdks/python/apache_beam/transforms/periodicsequence.py

```diff
@@ -81,6 +86,9 @@ def process(
     '''
     start, _, interval = element
 
+    if isinstance(start, Timestamp):
```

Comment:
Not directly related to your change. But this code has some other issues.

`current_output_timestamp` is not a Timestamp and needs to be converted later.
mixed use of Timestamp and timestamp.Timestamp - we can pick either.